lib/url_generator.js

```diff
@@ -230,6 +230,10 @@ function getLocalFile (sidebar, href, source, fullUrl) {
 function validateLocalFile (sidebar, href, source, render) {
   const hash = extractHash(href)
 
+  if (href.indexOf('changelog') !== -1) {
```

Comment:
It's safe to use `String.prototype.includes` nowadays.

https://node.green/#ES2015-built-in-extensions-String-prototype-methods-String-prototype-includes

`if (href.includes('changelog'))`